.github/workflows/ci.yaml

```diff
@@ -48,11 +48,13 @@ jobs:
           - 2.6
           - 2.7
           - 3.0
```

Comment:
You have to take versions in quoutes: https://github.com/ruby/setup-ruby#matrix-of-ruby-versions

> Due to https://github.com/actions/runner/issues/849, we have to use quotes for '3.0'

Reply:
@AlexWayfer done! Didn't notice it. Thanks for pointing this out.